Frameworks/UIKit/NSParagraphStyle.mm

```diff
@@ -21,12 +21,17 @@
 #import "NSParagraphStyleInternal.h"
 
 @implementation NSParagraphStyle
+
+- (void)dealloc {
+    _tabStops = nil;
```

Comment:
!NO!

If this file is not ARC-enabled, and _tabStops is not a `StrongId` (it's a property's backing ivar, so it's not likely to be a `StrongId`), you must *-release* it.